recipe/meta.yaml

```diff
@@ -14,7 +14,7 @@ source:
 build:
   number: 0
   script: python setup.py install --single-version-externally-managed --record record.txt
-  skip: True  # [win or not py35]
+  skip: True  # [win]
```

Comment:
> - [x] This should be # [win or not py>=35] since it is a recommendation. 

> - [x] Last but but least this need to be re-rendered to get the changes in.

Thanks for pointing out the rerendering instructions, I had been following the process outlined on https://conda-forge.github.io/#contribute .